.obs/dockerfile/slem4r-base-iso/Dockerfile

```diff
@@ -24,17 +26,18 @@ COPY --from=builder /output /elemental-iso
 
 ARG SLEMICRO_VERSION
 ARG BUILD_REPO=%%IMG_REPO%%
-ARG IMAGE_REPO=$BUILD_REPO/suse/sle-micro-rancher-iso/base-$SLEMICRO_VERSION
+ARG IMAGE_REPO=$BUILD_REPO/suse/sle-micro-iso/base-$SLEMICRO_VERSION
+ARG IMAGE_TAG=%VERSION%-%RELEASE%
 
 # Define labels according to https://en.opensuse.org/Building_derived_containers
 # labelprefix=com.rancher.slem4r
 LABEL org.opencontainers.image.title="SLE Micro for Rancher base-ISO"
 LABEL org.opencontainers.image.description="Includes the SLE Micro for Rancher ISO"
-LABEL org.opencontainers.image.version="%VERSION%"
+LABEL org.opencontainers.image.version="${IMAGE_TAG}"
```

Comment:
I like keeping this version here equal to the most specific tag. Reason for that is that IMHO makes everything clear as this version can be read as a tag in the registry and it should be easy to identify a git tag relating to that in sources.

I have no strong opinion on what `%VERSION%` includes as long as:
* this is what we use to tag in our git repo
* it can be sorted and compared
* we can bump it any time so we can tag and ship a new version including changes in our code (e.g. Dockerfile changes) without having to wait for a new underlying OS version.